tools/autograd/derivatives.yaml

```diff
@@ -1903,43 +1903,52 @@
   self: replication_pad3d_backward(grad, self, padding)
   result: auto_linear
 
-  # NOTE: Not implementing forward AD formulas for non-vec upsample overloads because they are
-  #       only kept for backward compatability
```

Comment:
This is discussed at https://github.com/pytorch/pytorch/pull/70864/files#r789936888 apparently